src/policy-store.ts

```diff
@@ -83,7 +82,7 @@ export interface PolicyStoreProps {
    *
    * @default - No schema.
    */
-  readonly schema?: ISchema;
+  readonly schema?: cedar.Schema;
```

Comment:
i think we should stick with the properties as we defined before and keeping the validation as an option in the constructor of Policy Store Method when we build a static policy (maybe should it be optional?). This is to adhere to [CDK Guidelines](https://github.com/aws/aws-cdk/blob/main/docs/DESIGN_GUIDELINES.md#props) ( see “props” and “types” sections we must adhere to them if we want the construct to be merged inside official cdk library in future).

Reply:
another big reason: we can't expose non-jsii compatible classes